src/containers/my-courses/course-search-tool-bar/CourseSearchToolbar.style.ts

```diff
@@ -0,0 +1,94 @@
+import { TypographyVariantEnum } from '~/types'
+
+export const styles = {
+  container: {
+    minHeight: '110px',
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'space-between'
+  },
+  autocomplete: {
+    width: '100%',
+    maxWidth: { md: '370px' },
+    mr: '30px',
+    '& .MuiOutlinedInput-root': {
+      padding: '5px 9px'
+    },
+    label: {
+      lineHeight: '20px'
+    },
+    position: 'relative',
+    '& .MuiFormHelperText-root': {
+      fontSize: '14px',
+      position: 'absolute',
+      top: '-33px',
+      left: '-14px'
+    }
+  },
+  drowlevel: {
+    width: '370px',
+    position: 'relative'
+  },
+  drowstyle: {
+    fontSize: '14px',
+    position: 'absolute',
+    top: '-33px',
+    left: '-14px'
```

Comment:
you are using same style as for .MuiFormHelperText-root, so create an object above with these properties and use spread